src/components/MenuItem.js

```diff
@@ -10,6 +10,10 @@ import Icon from './Icon';
 import {ArrowRight} from './Icon/Expensicons';
 
 const propTypes = {
+    // Any additional styles to apply
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.any,
```

Comment:
Oh, right. Good point. Yea, initially I called it additionalStyles, which I think I'll go back to. thanks 👍

Reply:
I kind of like the pattern of naming the prop after the style it's for e.g. since this is the pressable which is acting like the wrapper maybe this can be `wrapperStyle`. Which sort of set us up well if we have a `contentStyle` down the line.